fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryPlanWithIndex.java

```diff
@@ -63,6 +65,14 @@ public interface RecordQueryPlanWithIndex extends RecordQueryPlan {
     @Nonnull
     IndexScanType getScanType();
 
+    default ScanComparisons getComparisons() {
+        return null;
+    }
+
+    default KeyExpression getCommonPrimaryKey() {
+        return null;
```

Comment:
Do query plans other than `RecordQueryIndexPlan` need these two methods, or are there times that code has a `RecordQueryPlanWithIndex` and needs to call `getComparisons` or `getCommonPrimaryKey`?

Reply:
The `getComparisons` method was removed from the interface (per Mike's comment below).
Also removed the `getCommonPrimaryKey` as it is not needed at the interface level (was a leftover from a previous necessity).